src/platform-implementation-js/views/thread-row-view.ts

```diff
@@ -15,6 +15,7 @@ export interface ImageDescriptor {
   imageClass?: string;
   tooltip?: string;
   orderHint?: number;
+  onHover?: (event: { hoverEnd: Promise<void> }) => void;
```

Comment:
```suggestion
  /** @internal */
  onHover?: (event: { hoverEnd: Promise<void> }) => void;
```

Reply:
I think it's fine to expose it regularly. It's a simple API that doesn't need to change much and if we had to drop support for it, it wouldn't necessarily break applications using the live-loaded version.